README.md

```diff
@@ -184,3 +184,26 @@ Formats are ways to present notebooks to the user.
 If you'd like to write a new format, open a ticket, or speak up on [gitter][]!
 We have some work yet to do to support your next big thing in notebook
 publishing, and we'd love to here from you.
+
+## Securing the Notebook Viewer
+
+You can run the viewer as a JupyterHub 0.7+ service. Doing so prevents users who have not authenticated with the Hub from viewing any notebooks. This setup can be useful for protecting access to local notebooks rendered with the `--localfiles` option.
```

Comment:
Let's go ahead and link to the JupyterHub services section: http://jupyterhub.readthedocs.io/en/latest/services.html

Also:
~~Doing so~~``Running the viewer as a service prevents users who have not authenticated with the Hub...``